Jonathan-Le.txt

```diff
@@ -0,0 +1,222 @@
+    
+    An h1 tag is use for a heading on a title. Only once is needed.
+    
+    5. Name two tags that have required attributes
+   
+    The 'img' tag and 'audio' tag.
+
+    6. What do we put in the head of our HTML document?
+ 
+    The 'meta' tags and title tag.
+
+    7. What is an id? 
+
+    A id is a unique attribute gives a specific information to a tag.
+
+    8. What elements can I add an id to? 
+
+    The 'a' tag, 'header' tag, 'section' tag, h1 - h6 tags and 'div' tags. 
+     
+    9. How many times can I use the same id on a page? 
+    
+    Only one time.
+
+    10. What is a class?
+    
+    A class is an attribute that adds more information to an element.
+
+    11. What elements can I add a class to? 
+
+    A 'section' tag, 'div' tag, 'header' tag, 'p' tag, 'h1' to 'h6' tag, 'ol' tag, 'ul' tag,
+    'table' tag, 'span' tag and 'li' tag.
+ 
+    12. How many times can I use the same class on a page?
+ 
+    Many times we want.
+
+    13. How do I get my link to open in a new tab?
+
+    target = _blank
+
+    14. What is the alt attribute used for? 
+
+    It gives a text name from the image when an image is not displayed.
+
+    15. How do I reference an id?
+    
+    <element = id = "id">
+
+    16. What is the difference between a section and a div
+
+    A 'section' is semantic and a 'div' is non-semantic.
+
+    17. What is CSS used for? 
+
+    CSS is use for styling a webpage.
+
+    18. How to we select an element? Example - every h2 on the page
+
+    h2 {
+
+    }
+    
+    19. What is the difference between a class and an id? - Give me an example of when I might use each one
+    
+    A class is an attribute used to select elements with the same attributes. An id is an attribute use to specify
+    a unique element. 
+    
+    <html>
+    <body>
+    <head>
+    <title> Name of webpage </title>
+    <h1>Food page</h1>
+    <header id = "main">There are all different kinds of foods we can learn about.</header>
+    <section class = "Fruits">
+    <li>
+    <h2 class = "Food">Apples</h2>
+    <p></p>
+    <h2 class = "Food">Oranges</h2>
+    <p></p>
+    <h2 class = "Food">Pears</h2>
+    <p></p>
+    </li>
+    </section>
+    <section class = "Veggies">
+    <li>
+    <h2 class = "Food">Lettuce</h2>
+    <p></p>
+    </li>
+    </section>
+    </body>
+    </html>
+
+    20. How do we select classes in CSS?
+    
+    The '.' key.
+
+    21. How do we select a p element with a single class of “human””?
+
+    " p .human ".
+
+    22. What is a parent child selector? When would this be useful? 
+
+    It's a descendant selector. A descendant has a space between the two selectors.
+    A parent selector with a child selector nested from the parent. Using a descendant 
+    selector would be useful when there's another class attribute nested in a class attribute.
+
+    23. How do you select all links within a div with the class of sidebar?
+    
+    div .sidebar {
+     
+    }
+

```

Comment:
`div .sidebar` would select all elements with a `.sidebar` class inside a `<div>` element.  The HTML element for links is `<a>`.  So you would use div.sidebar a